package.json

```diff
@@ -24,5 +25,9 @@
     "lodash.mapvalues": "^4.6.0",
     "mocha": "3.0.2",
     "prettier": "1.7.0"
-  }
+  },
+  "browserslist": [
+    "last 2 versions",
+    "IE 11"
+  ]
```

Comment:
Will this get picked up by consumer projects too or will they still have to add this manually?